src/clientdlgbase.ui

```diff
@@ -582,7 +593,7 @@
   <tabstop>rbtReverbSelR</tabstop>
  </tabstops>
  <resources>
-  <include location="resources.qrc"/>
+  <include location="../../jamulus3_8_0rc1/src/resources.qrc"/>
```

Comment:
Same

Reply:
I have no explanation how that got in there.  I checking to correct.

Reply:
Those lines do not exist in my source.
I do not know what to do.
I have pushed again with small changes to the UI to assure the display of >500 etc.
I hope these errors are now gone.

Reply:
>  I hope these errors are now gone.
Looks good!

(Not marking this as resolved yet as it contains a hint for @softins regarding the 500ms question above)